py/desispec/trace_shifts.py

```diff
@@ -491,9 +491,6 @@ def compute_dx_from_cross_dispersion_profiles(xcoef,ycoef,wavemin,wavemax, image
 
     t0=time.time()
 
-    if fibers is None :
-        fibers = np.arange(psf.nspec)
-
```

Comment:
I think the correct change here is to set the default `fibers=None` in the function definition, not remove this logic.  `fibers` is an input to `compute_dx_from_cross_dispersion_profiles`, and if the user provides `fibers=None` then it is supposed to default to all fibers.

Reply:
The problem is that PSF is not defined. I couldn't quickly figure out where to get nfibers from if not psf

Reply:
Whoops! Indeed. I corrected this to be
```python
    if fibers is None :
        fibers = np.arange(xcoef.shape[0])
```
since `xcoef` is a 2D array of shape `(nspec, ncoef)`.